app/templates/components/orange-bar-data-explorer.hbs

```diff
@@ -20,6 +20,7 @@
 
       <Explorer::ModifyTablesMenu
         @mode={{@mode}}
+        @sources={{@sources}}
```

Comment:
This works! In this case I think it's better to just pass in an extra argument of `@source` to `OrangeBarDataExplorer` and  down to  `ModifyTablesMenu`. Then  in `modify-tables-menu.js` you  don't have to duplicate code  to find the selected source. Might be a small advantage later if we ever refactor the `.selected` property.